lib/Input.js

```diff
@@ -127,20 +127,18 @@ class Mouse {
    * @param {number} y
    * @param {!Object=} options
    */
-  async click(x, y, options) {
+  async click(x, y, options = {}) {
     this.move(x, y);
     this.down(options);
-    if (options && options.delay)
+    if (typeof options.delay === 'number')
```

Comment:
Ack. Kinda pointless to pass `0` but I figured we should support it. I can change it back if you want.